chat-client/src/client/mynahUi.ts

```diff
@@ -144,6 +150,7 @@ export const createMynahUi = (messager: Messager, tabFactory: TabFactory): [Myna
             messager.onTabAdd(tabId)
             const defaultTabConfig: Partial<MynahUIDataModel> = {
```

Comment:
I suspect that this is not gonna work as expected. Below, we're setting the tab defaults, and while doing that depending on the initial value of the disclaimerAcknowledged argument we're adding the disclaimer part. 
This is set to all tabs which will open in future too. 

In here, depending on the situation we're adding or not adding the disclaimer card. But NOT adding means that we're also not touching the defaults. And defaults still contain the disclaimer card.

What I would prefer would be not playing with the defaults and not setting the disclaimerCard to defaults at all. 

Do it in the initial `tabs`, do it in the `onTabAdd`, but not in the `defaults`. We don't need to write it to defaults since we're checking the statement when a new tab is open and adding it depending on the statement to that tab.

Reply:
Resolved, thanks.